ingestion/examples/sample_data/datasets/tables.json

```diff
@@ -3,7 +3,6 @@
     {
       "id": "3cda8ecb-f4c6-4ed4-8506-abe965b54b86",
       "name": "dim_address",
-      "fullyQualifiedName": "bigquery_gcp.shopify.dim_address",
```

Comment:
@harshach, we have removed the `fullyQualifiedName`, is this change is intended?

Reply:
Hey Sachin - this shouldn't affect UI - Ingestion will create a FQN using FQN separator itself.

Reply:
this is not needed for ingestion to carry-forward. Backend will ignore any FQN being sent as part of createTableRequest and add its own